kotlin/internal/jvm/compile.bzl

```diff
@@ -653,7 +653,7 @@ def _run_kt_java_builder_actions(
             associates = associates,
             compile_deps = compile_deps,
             deps_artifacts = deps_artifacts,
-            annotation_processors = [],
+            annotation_processors = annotation_processors,
```

Comment:
Pure hacks to run KSP under the compilation pass of Kotlinc instead of the Kapt pass.